projects/storefrontstyles/scss/themes/calydon/components/checkout/steps/shipping-method/_delivery-mode.scss

```diff
@@ -0,0 +1,15 @@
+%cx-delivery-mode-calydon {
```

Comment:
Everything looks fine. If we add new radio buttons in other places the fix won't propagate. Should we make this a more global fix.

Also if you go to the consent page the check boxes have the same issue on firefox and the fix would be identical. Is there a ticket for this?